libmisc/obscure.c

```diff
@@ -170,8 +119,8 @@ static /*@observer@*//*@null@*/const char *password_check (
 		msg = _("case changes only");
 	} else if (similar (oldmono, newmono)) {
 		msg = _("too similar");
-	} else if (simple (old, new)) {
-		msg = _("too simple");
+	} else if (strlen (new) < 8) {
```

Comment:
this is more restrictive than the existing code.  having it be dynamic based on the inputs was aligned with the intention of this code.  but you're making it static, and we already have `PASS_MIN_LEN` as a configurable value that is checked earlier.  so if you want to delete the dynamic strength checks, you should delete this length check entirely.

not that i have a problem with the dynamic strength check as-is.

Reply:
> this is more restrictive than the existing code. having it be dynamic based on the inputs was aligned with the intention of this code. but you're making it static, and we already have `PASS_MIN_LEN` as a configurable value that is checked earlier. so if you want to delete the dynamic strength checks, you should delete this length check entirely.

Yep, I didn't see the separate `PASS_MIN_LEN` test.  Then I'll just remove this test.

> 
> not that i have a problem with the dynamic strength check as-is.

It rewards passwords using wider character sets by allowing them to be shorter, which can reinforce the idea that it's a good thing to do in general.  Again, if one can remember long passwords using symbols and uppers, that's great, but enforcing that is problematic, and even this kind of reinforcement I believe it to be counterproductive in that regard.